examples/gno.land/p/demo/tests/subtests/subtests.gno

```diff
@@ -0,0 +1,17 @@
+package subtests
+
+import (
+	"std"
+)
+
+func CurrentRealmPath() string {
+	return std.CurrentRealmPath()
```

Comment:
why not also make this CurrentRealm()?

Reply:
This PR introduce `std.PrevRealm` not `std.Realm`, this will come in a next PR.